style_editor.html

```diff
@@ -9525,7 +9560,7 @@
 /* Variant Slider functions */
 
 function updateVariantValue(newValue) {
```

Comment:
I realize the computational requirements are basically none, but is there anything to be gained by taking the time to set a variable with a value of 0 to 0?

Reply:
If the entire value is deleted, an autofill of zero gets put in place to avoid a null value from being left there. Since neither the slider nor the plus or minus buttons nor a key press of the digit 0 caused the zero, this needs to be here.

Reply:
Ah, so you're using this as a loosely equals check.  Might be worth noting that in a code comment.

Reply:
Comment added